project-base/storefront/components/Forms/Radiobutton/Radiobutton.tsx

```diff
@@ -46,7 +43,7 @@ export const Radiobutton = forwardRef<HTMLInputElement, RadiobuttonProps>(
                     value={value}
                     onBlur={onBlur}
                     onChange={onChange}
-                    onClick={onClickHandler}
+                    onClick={onClickWithNull}
```

Comment:
I don't understand the motivation behind using this naming instead of general like `onClickHandler`. Here I don't get why it's necessary to expose to naming that function works with `null` value? Do you have some new mindset for naming those handlers?

Reply:
No, not really. I will just change it back.